state-chain/traits/src/mocks/block_height_provider.rs

```diff
@@ -19,6 +19,13 @@ impl<C: Chain> BlockHeightProvider<C> {
 	pub fn set_block_height(height: C::ChainBlockNumber) {
 		Self::put_value(BLOCK_HEIGHT_KEY, height);
 	}
+
+	pub fn mutate_block_height<F>(f: F)
```

Comment:
we only ever use this to increment - we could just simplify and make this "increment" instead

Reply:
Changed to `increment_block_height()`.